### PR TITLE
update style css

### PR DESCRIPTION
**Pull request by mat-witts**
added css quotes property with double quotes as standard and single quotes if embedded including entity numbers in comments for more style options

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -22,7 +22,21 @@
 }
 
 /* q */
-.dokuwiki q.blockquote-plugin {
+.dokuwiki q.blockquote-plugin {\0022
```
Comment (Klap-in) [inline style.css]: Is the entity on this line intended?

**Comment by LarsGit223** [comment]
What are the changes doing? I tested in a firefox browser and did not see any effect.
(I tested this code: ```<q>This is a <q>quote</q> inside a quote.</q>```)

File: style.css (hunk 1 above)
Comment (mat-witts) [inline style.css]: this gives a default behavior of double quotes for first level quotes tag, then second level (embeded) renders as single quotes but you could change this behavior using any of the entity codes given in the comment. This standard is documented [here](https://www.w3schools.com/CSSref/pr_gen_quotes.asp)

Comment (LarsGit223) [inline style.css]: If I leave the **\0022** entity in the CSS code I get an error in DokuWiki ("A fatal error occurred during compilation of the CSS files..."). IMHO that has definitely to be removed. So I did all my tests without that entity.

**Comment by LarsGit223** [comment]
If I test this code:
```
Plain text <q>This is a <q>quote</q> inside a quote.</q> plain text
```
then it looks like this:
blockquote

The blockquote plugin does not seem to show any quotes. I tested with Frusterick Manners, actual Blockquote plugin, standard DokuWiki template. Firefox and Chrome on my Ubuntu machine show the same results.

Could you post an image of how it looks on your installation? Am I doing something wrong?

**Comment by LarsGit223** [comment]
I also saw the documentation at w3schools. I also think that setting the property is correct (the line ```quotes: "\0022" "\0022" "\0027" "\0027";```) :thumbsup:

It just has no visual effect on my installation. So I would like to see how it looks like for you.

**Comment by mat-witts** [comment]
using the following line:

`quotes: "\0027" "\0027" "\2039" "\203A";`

the result looks like this in FF53.03 (single quotes with embeded single left and right angle entity code:

screenshot from 2017-06-04 16-13-49

**Comment by LarsGit223** [comment]
Hmmm...strange. Thanks for the screenshot. Which versions, browser and template are you using?

File: style.css [hunk 2]
```diff
@@ -25,4 +25,18 @@
 .dokuwiki q.blockquote-plugin {
     border: 1px dotted __border__;
     background: __background_alt__;
+    quotes: "\0022" "\0022" "\0027" "\0027";
+    /*
+" 	double quote 	\0022
```
Comment (LarsGit223) [inline style.css]: Would you mind removing this large comment block? Reminders are nice but this one's too large in my opinion.

**Comment by LarsGit223** [comment]
Ok, if I switch to the **bootstrap3** template then it looks like your screenshot is showing it!

**Comment by LarsGit223** [comment]
I tested some more templates:
This ones do not show any quotes:
- dokuwiki
- white
- writr
- zenith

This ones show quotes:
- bootstrap3
- dokubootstrapsimplified
- wallpaper

So it's only a template thing. If you remove that last large comment I'll be glad to merge your PR.

**Comment by mat-witts** [comment]
replaced long comment with short one... maybe provide info. on variation of behavior with templates on docs?

**Comment by LarsGit223** [comment]
Thanks looks good.

> replaced long comment with short one... maybe provide info. on variation of behavior with templates on docs?

I think a little remark on the plugin page would be enough?

**Comment by LarsGit223** [comment]
Thanks for your contribution!

**Pull request merged**